harness/determined/horovod.py

```diff
@@ -121,6 +121,23 @@ def create_performance_args(env: det.EnvContext) -> List[str]:
     return performance_args
 
 
+def create_horovod_timeline_args(env: det.EnvContext) -> List[str]:
+    optimizations = env.experiment_config.get("optimizations", {})
+    check.check_in("horovod_timeline", optimizations)
+
+    htimeline_path = optimizations.get("horovod_timeline")
+    if htimeline_path is None:
+        return []
+
+    return [
+        "--timeline-filename",
+        htimeline_path,
+        "--timeline-mark-cycles" # This is useful information, but it is strange that it is not
```

Comment:
non-blocking: this format is a bit strange, could we put the comment b/w the lines?